src/Command/Pull/PullDatabaseCommand.php

```diff
@@ -40,8 +42,13 @@ protected function configure() {
    */
   protected function execute(InputInterface $input, OutputInterface $output) {
     parent::execute($input, $output);
-    $this->pullDatabase($input, $output);
-    if (!$input->getOption('no-scripts')) {
+    $no_scripts = $input->hasOption('no-scripts') && $input->getOption('no-scripts');
```

Comment:
I converted these to named parameters since they now interact (download implies no-scripts), and the whole `hasOption() && !getOption()` pattern is a little unintuitive to begin with.